azext_edge/edge/_help.py

```diff
@@ -650,6 +650,69 @@ def load_iotops_help():
         short-summary: Dataflow endpoint management.
     """
 
+    helps[
+        "iot ops dataflow endpoint apply"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow endpoint resource.
+        long-summary: |
+          An example of the config file format is as follows:
+
+          ```
+          {
+            "properties": {
```

Comment:
User shouldn't have to include properties since its implied.

Reply:
I see, looks like both with or without properties would work for `get_file_config`, will update!